poglink/cogs/rates.py

```diff
@@ -105,9 +109,15 @@ async def send_embed(self, description, url):
 
         server_name = server_meta.get("short_name")
 
+        if title is None:
+            # generate dynamic timestamp (https://hammertime.djdavid98.art/)
+            ts = int(datetime.datetime.now().timestamp() // 60 * 60)
+            ts_string = f"<t:{ts}:t>"
+            title = f"ARK's {server_name} server rates updated at {ts_string}"
+
```

Comment:
This allows for the title to be provided externally but still fall back to the same default.